run_build.ps1

```diff
@@ -49,7 +45,7 @@ cat "$RepoRoot\branchinfo.txt" | ForEach-Object {
 # Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot
 if (!$env:DOTNET_INSTALL_DIR)
 {
-    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\Windows\$Architecture"
+    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\$Architecture"
```

Comment:
Can you update the `update-dependencies.ps1` script to also point to the same "Stage0" location?  It should point to the same place, to avoid downloading stage0 twice in different locations.